includes/class-wc-payments-fraud-service.php

```diff
@@ -188,4 +207,24 @@ private function get_cookie_session_id() {
 		$cookie_customer_id = $cookie[0];
 		return $wpcom_blog_id . '_' . $cookie_customer_id;
 	}
+
+	/**
+	 * If a "forterToken" cookie is present, send it to the WCPay server so the
+	 * current browsing session can be linked to the account. It will only be sent once.
+	 */
+	public function send_forter_cookie_token() {
+		if ( ! $this->account->is_stripe_connected() || ! isset( $_COOKIE['forterToken'] ) ) {
+			return;
+		}
+
+		$account_id = $this->account->get_stripe_account_id();
+		if ( get_option( 'wcpay_forter_token_sent' ) !== $account_id ) {
+			// The cookie contents are opaque to us, so it's better to not sanitize them.
+			//phpcs:ignore WordPress.Security.ValidatedSanitizedInput
+			$response = $this->payments_api_client->send_forter_token( $_COOKIE['forterToken'] );
```

Comment:
If an exception is thrown by this function the entire site crashes, we should probably wrap this in a try/catch and log the error.

Reply:
Fixed in https://github.com/Automattic/woocommerce-payments/pull/1349/commits/10780f2f3758e1a0c751c9a97b5be7f24bb7ec3e